src/main/java/org/folio/circulation/domain/policy/LoanPolicy.java

```diff
@@ -57,17 +57,8 @@
   private LoanPolicy(JsonObject representation) {
     this(representation,
       new NoFixedDueDateSchedules(),
-      new NoFixedDueDateSchedules());
-  }
-
-  LoanPolicy(
-    JsonObject representation,
-    FixedDueDateSchedules fixedDueDateSchedules,
-    FixedDueDateSchedules alternateRenewalFixedDueDateSchedules) {
-    super(getProperty(representation, "id"), getProperty(representation, "name"));
-    this.representation = representation;
-    this.fixedDueDateSchedules = fixedDueDateSchedules;
-    this.alternateRenewalFixedDueDateSchedules = alternateRenewalFixedDueDateSchedules;
+      new NoFixedDueDateSchedules(),
```

Comment:
Can this be included on the previous line? I don't think the parameters need to be split one per line (that is likely a hangover from a style I used to use and probably should start moving away from)